kedro/framework/session/session.py

```diff
@@ -209,6 +210,7 @@ def _setup_logging(self) -> None:
         """Register logging specified in logging directory."""
         conf_logging = self._get_logging_config()
         logging.config.dictConfig(conf_logging)
```

Comment:
This needs to move inside `configure_logging()`.

Reply:
Do you mean `logging.config.dictConfig(conf_logging)` has to be called in `configure_logging()`?